tests/test_preload_script.py

```diff
@@ -95,3 +389,152 @@ async def test_addPreloadScript_logging(websocket, context_id, html):
     # Assert navigation is finished.
     result = await read_JSON_message(websocket)
     assert result == {"id": command_id, "result": ANY_DICT}
+
+    # Create a new iframe within the same context.
+    result = await execute_command(
+        websocket, {
+            "method": "script.evaluate",
+            "params": {
+                "expression": f"""const iframe = document.createElement('iframe');
+                    iframe.src = `{html('<script>console.log("I am the child");</script>')}`;
+                    document.body.appendChild(iframe);""",
+                "target": {
+                    "context": context_id
+                },
+                "awaitPromise": True,
+                "resultOwnership": "root"
+            }
+        })
+
+    # Asset that the preload script is executed in the new iframe.
+    result = await read_JSON_message(websocket)
+    assert result == AnyExtending({
+        "method": "log.entryAdded",
+        "params": {
+            "args": [{
+                "type": "string",
+                "value": "my preload script"
+            }]
+        }
+    })
+
+    # Asset that the child script is executed last.
+    result = await read_JSON_message(websocket)
+    assert result == AnyExtending({
+        "method": "log.entryAdded",
+        "params": {
+            "args": [{
+                "type": "string",
+                "value": "I am the child"
+            }]
+        }
+    })
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("subscribe_to_context", [True, False])
+async def test_addPreloadScriptGlobally_loadedInMultipleContexts(
```

Comment:
test name and parameter are controversial

Reply:
Do you have a specific suggestion?

Reply:
e.g. you can merge it with the previous test and rely on the events or both on event and global property

Reply:
Removed `Globally` from method name